packages/block-editor/src/components/block-mover/index.js

```diff
@@ -86,7 +86,7 @@ export class BlockMover extends Component {
 		// to an unfocused state (body as active element) without firing blur on,
 		// the rendering parent, leaving it unable to react to focus out.
 		return (
-			<div className={ classnames( 'block-editor-block-mover', { 'is-visible': isFocused || ! isHidden, 'is-horizontal': orientation === 'horizontal' } ) }>
+			<Toolbar className={ classnames( 'block-editor-block-mover', { 'is-visible': isFocused || ! isHidden, 'is-horizontal': orientation === 'horizontal' } ) }>
```

Comment:
What impact does this change have? I think lately we favor GroupToolbar.

Reply:
I changed it now to `ToolbarGroup`